WeakAuras/Types.lua

```diff
@@ -1302,6 +1302,13 @@ WeakAuras.group_aura_stack_info_types = {
   stack = L["Aura Stack"]
 }
 
+WeakAuras.group_roles = { 
```

Comment:
we already have role_types, can we re-use and maybe add "any" to it?

Reply:
I thought about that but we don't actually want it to show "any" in the load conditions.